Classes/Command/AppCommandController.php

```diff
@@ -32,13 +33,10 @@ class AppCommandController extends CommandController
     /**
      * @Flow\Inject
      *
-     * @var SystemLoggerInterface
+     * @var ThrowableStorageInterface
```

Comment:
We have to pin this to the latest flow then, right?

Reply:
`ThrowableStorageInterface` was introduced in Flow 4.3, so this version would be sufficient

Reply:
Ah, had in mind it was later. Than it should be good to go except missing documentation